publish/publish.proj

```diff
@@ -132,6 +132,24 @@
     <Exec Condition="'@(_DownloadedSymbolsPackages)' != ''" Command="$(NuGetPushSymbolsCommand) %(_DownloadedSymbolsPackages.Identity)" />
   </Target>
   
+  <Target Name="PublishDebToolPackagesToFeed"
+          DependsOnTargets="CheckIfAllBuildsHavePublished"
```

Comment:
Chatted with @karajas about this.  He's going to look into how to enable this for a specific leg instead of a part of finalization.